wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/main/MainScreen.kt

```diff
@@ -159,10 +161,22 @@ fun MainScreenContent(
     cardViewModel: CardViewModel,
     scope: CoroutineScope,
     drawerState: DrawerState,
-    permissionTracker: PermissionTracker
+    permissionTracker: PermissionTracker,
+    capabilities: AndroidKeystoreSecureArea.Capabilities,
 ) {
     val snackbarHostState = remember { SnackbarHostState() }
     val haveRequiredPermissions = permissionTracker.granted(blePermissions)
+    var showLskfWarning by remember { mutableStateOf(false) }
+
+    if (showLskfWarning) {
+        WarningDialog(
+            onDismissRequest = { showLskfWarning = false },
+            dismissBtnText  = stringResource(R.string.qr_lskf_warning_dismiss_btn),
+            dialogTitle = stringResource(R.string.qr_lskf_warning_dialog_title),
+            dialogText = stringResource(R.string.qr_lskf_warning_dialog_text),
+        )
```

Comment:
Just use `AlertDialog` directly here, will be simpler to read since most readers know what that is and don't know what a `WarningDialog` is.